docs/ibc/overview.md

```diff
@@ -1,180 +1,150 @@
 
-For this reason, most modules should use UNORDERED channels as they require less liveness guarantees to function effectively for users of that channel.
+For this reason, most modules that use UNORDERED channels are recommended as they require less liveness guarantees to function effectively for users of that channel.
 
 ### [Acknowledgements](https://github.com/cosmos/cosmos-sdk/tree/master/x/ibc/core/04-channel)
 
-Modules may also choose to write application-specific acknowledgements upon processing a packet. This may either be done synchronously on `OnRecvPacket`, if the module processes packets as soon as they are received from IBC module. Or they may be done asynchronously if module processes packets at some later point after receiving the packet.
+Modules also write application-specific acknowledgements when processing a packet. Acknowledgements can be done:
+
+- Synchronously on `OnRecvPacket` if the module processes packets as soon as they are received from IBC module.
+
+- Asynchronously if module processes packets at some later point after receiving the packet.
 
-Regardless, this acknowledgement data is opaque to IBC much like the packet `Data` and will be treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgemnet in such a way that the sender module can decode it correctly. This should be decided through version negotiation during the channel handshake.
+This acknowledgement data is opaque to IBC much like the packet `Data` and is treated by IBC as a simple byte string `[]byte`. The receiver modules must encode their acknowledgement so that the sender module can decode it correctly and decided through version negotiation during the channel handshake.
 
-The acknowledgement may encode whether the packet processing succeeded or failed, along with additional information that will allow the sender module to take appropriate action.
+The acknowledgement can encode whether the packet processing succeeded or failed, along with additional information that allows the sender module to take appropriate action.
 
-Once the acknowledgement has been written by the receiving chain, a relayer will relay the acknowledgement back to the original sender module which will then execute application-specific acknowledgment logic using the contents of the acknowledgement. This may involve rolling back packet-send changes in the case of a failed acknowledgement (refunding senders).
+After the acknowledgement has been written by the receiving chain, a relayer relays the acknowledgement back to the original sender module which then executes application-specific acknowledgment logic using the contents of the acknowledgement. This acknowledgement can involve rolling back packet-send changes in the case of a failed acknowledgement (refunding senders).
```

Comment:
I think we mean "acknowledgement" in

> This acknowledgement can involve rolling back

Beware the unclear antecedent (what we technical writers call "the naked it" and "the naked this"). It's good practice to repeat the subject